integration-tests/tests/analysis-context-bar.js

```diff
@@ -1,17 +1,17 @@
 // This test is owned by the Interactive Analysis (IA) Team.
 const _ = require('lodash/fp')
-const { withRegisteredUser, withBilling, withWorkspace, performAnalysisTabSetup } = require('../utils/integration-helpers')
+const { deleteRuntimes, withWorkspace, performAnalysisTabSetup } = require('../utils/integration-helpers')
 const {
   click, clickable, getAnimatedDrawer, findElement, noSpinnersAfter, findButtonInDialogByAriaLabel
 } = require('../utils/integration-utils')
 const { registerTest } = require('../utils/jest-utils')
+const { withUserToken } = require('../utils/terra-sa-utils')
 
 
-const testAnalysisContextBarFn = _.flow(
+const testAnalysisContextBarFn = _.flowRight(
```

Comment:
Change this from `flow` to `flowRight` so that the order of arguments matches the order of operations in the test.